packages/web-app/src/components/common/InfoSection.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { styled } from '@mui/material/styles';
+import { Box, Typography } from '@mui/material';
+
+const SectionWrapper = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(1)};
+  padding: ${({ theme }) => theme.spacing(2)};
+  border: 1px solid ${({ theme }) => theme.palette.divider};
```

Comment:
It is subjective, but I think it looks better without a border around the section

Reply:
Borders removed